Add code example for include option of AM::Serialization#serializable_hash

radanskoric opened the pull request:
ActiveModel::Serialization#serializable_hash code examples were not showcasing the `include`option at all so I extended the current example to include it.

radanskoric committed:
Add code example for include option of AM::Serialization#serializable_hash

rafaelfranca commented on activemodel/lib/active_model/serialization.rb:
```diff
@@ -72,10 +72,20 @@ module ActiveModel
+    #     attr_accessor :city, :street
+    #
+    #     def attributes
+    #       {'city' => nil, 'street' => nil}
```
should not be `city` instead of `nil`?

radanskoric replied:
Not according to all the rest of the documentation in this class. All the other examples just use `=> nil` everywhere.

In fact, the values are completely irrelevant. `attributes` is used in only [one place](https://github.com/rails/rails/blob/master/activemodel/lib/active_model/serialization.rb#L100) and that is only to extract the keys.

As far as I can tell, the only reason a Hash is expected here instead of an array of keys is so it would have the same requirements as `ActiveModel::AttributeMethods`.

rafaelfranca commented:
Applied at 3d949f34816d6eca0a6b59cfa08d91f36e8e64dd. Thanks

radanskoric commented:
Hey, @rafaelfranca thanks, but I killed this PR because, while it was open, a different PR was opened and merged so there is now a longer explanation in place and this PR was no longer relevant: https://github.com/rails/rails/blob/master/activemodel/lib/active_model/serialization.rb#L111-L140

So what you merged is just some example preparation code without actual usage example (since I noticed the other commit only after I rebased and cleaned up the conflict) which makes it a bit confusing. So sadly I think this commit should be reverted. :(
